browser/src/map/handler/Map.Keyboard.js

```diff
@@ -971,13 +970,13 @@ L.Map.Keyboard = L.Handler.extend({
 
 	readOnlyAllowedShortcuts: function(e) {
 		// Open keyboard shortcuts help page
-		if (this._isCtrlKey(e) && e.shiftKey && e.key === '?')
+		if (this._isCtrlKey(e) && e.shiftKey && e.keyCode === this.keyCodes.FORWARDSLASH) // ?
```

Comment:
This seems to not work with Turkish keyboard layout. Please revert the following occurrences of this line
```javascript
if (this._isCtrlKey(e) && e.shiftKey && e.keyCode === this.keyCodes.FORWARDSLASH) { // ?
``` 
and then we can merge this PR :)

Nevertheless, I do agree, we could have a better shortcut for the keyboard shortcuts dialog. We need to think and test this further with multiple different keyboard layouts

Thank you so much for not only testing it deeply but also helping out !

Reply:
Thank you for investigating the matter. I concur that it's more accurate to retain the link to the char '?'. I have made the necessary changes to the code.

Reply:
Thank you!